lib/index.js

```diff
@@ -267,11 +274,21 @@ export default {
             range = Helpers.generateRange(textEditor, line);
           }
 
+          // avoid duplicated issues
+          if (fingerprints.has(issue.fingerprint)) {
+            return;
+          }
+          fingerprints.add(issue.fingerprint);
+
           linterResults.push({
-            type: 'Warning',
-            text: issue.description,
-            filePath,
-            range,
+            severity: mapSeverity[issue.severity] || 'warning',
+            excerpt: `${issue.engine_name.toUpperCase()}: ${issue.description} [${issue.check_name}]`,
+            description: (issue.content && issue.content.body) ? issue.content.body : undefined,
+            url: encodeURI(`https:www.google.com/#q=${issue.engine_name} ${issue.check_name}`),
```

Comment:
I'd rather not generate a search term like this at all. If the linter doesn't give you a URL, or you can't generate an exact URL from the rule name (like is done [here](https://github.com/jfmengels/eslint-rule-documentation) for ESLint rules), then it shouldn't be included at all.

The issues with generating a search term is that then _every single_ message "has" a URL, and in this case it isn't providing much benefit as the use can just copy/paste from the message.